src/components/Sum.jsx

```diff
@@ -1 +1,13 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Sum({ a = 0, b = 0 }) {
```

Comment:
also, you can use the arrow function expression for function when you intermediate return a value